### ISSUE TITLE
Typo in w32/user32.go

### ISSUE THREAD

**Issue by njm227** [tadvi/winc#2]
There’s a typo in line 934 of w32/user32.go where it should be EndPaint instead of BeginPaint

**Comment by neurlang** [comment]
This is true. Can be fixed or start using the more higher level winc package directly

**Comment by neurlang** [comment]
Actually, this was fixed. Please close.
